src/features/modals/ExportDialog.tsx

```diff
@@ -1,15 +1,19 @@
+import { useCallback } from "react";
 import { Modal, Button } from "react-bootstrap";
+import { useAppDispatch } from "../../app/hooks";
+import { modalSlice } from "./modalSlice";
 
 interface ExportDialogProps {
-  onHide: () => void;
   show: boolean;
   exportText: string;
   handleSave: () => void;
 }
 
 export const ExportDialog: React.FC<ExportDialogProps> = (props) => {
+  const dispatch = useAppDispatch();
+  const hideModal = useCallback(() => dispatch(modalSlice.actions.hideAll()), []);
```

Comment:
Why use `useCallback` here if there are no dependencies?

Reply:
I suppose `dispatch` is a dependency, now that I look at it, eh?  I'll add that to the list.

Basically I didn't want the `hideModal()` callback to be re-generated on every render.